### PR TITLE
Create afk message

### PR DESCRIPTION
**Pull request by SupaHam**
Based on #590 

This commit makes it possible for players to set an afk message to detail their reasoning for being away to other players. This can be especially useful for the /msg command; instead of replying the user is afk, it will specify why they are afk so the sender can act further on that information.

Two methods have been added to IUser: `getAfkMessage()` and `setAfkMessage(String)`.

All locale files have two new messages: `userAFKWithMessage` and `userIsAwayWithMessage`. They all use the messages that do not have the suffix _WithMessage_, i.e. `userAFK` and `userIsAway`. Furthermore, the `userIsAwayWithMessage` will not utilise the second parameter, the reason, to prevent spam and unnecessary text in global chat. However, the second parameter ({1}) is available for use in `userIsAwayWithMessage`. `userAFKWithMessage`, which is sent to `/msg` senders, does use the {1} as it controllable spam by the command sender themselves.

`/afk` usage is now: `/afk [player/message...]`

### CODE REVIEW

File: Essentials/src/com/earth2me/essentials/IUser.java [hunk 1]
```diff
@@ -147,4 +147,8 @@
     CommandSource getSource();
 
     String getName();
+
```
Comment (drtshock) [inline Essentials/src/com/earth2me/essentials/IUser.java]: not to be too picky but empty line here vs spaces D:

**Comment by SupaHam** [comment]
Anything else that needs to be done?

**Comment by SupaHam** [comment]
@drtshock @vemacs

**Comment by vemacs** [comment]
Possible feature additions:
- Permission for AFK message
- Disabling this ability if muted (for precedent, see https://github.com/SupaHam/Essentials/blob/795ffec8b8ebea9bb1977bd593f609c0d368d538/Essentials/src/com/earth2me/essentials/commands/Commandme.java#L25)

**Pull request merged**